frontend/app/createApiClient.ts

```diff
@@ -1,9 +1,8 @@
 import { AppClient } from "./client";
 
-
 export function createApiClient(token?: string): AppClient {
-    return new AppClient({
-        BASE: "http://localhost:8000",
-        TOKEN: token,
-    });
-}
\ No newline at end of file
+  return new AppClient({
+    BASE: "http://localhost:8000",
```

Comment:
We need to put it in some config map or just change it using sed or other command in pipeline.

Reply:
I will change it to use env